cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -1,181 +0,0 @@
-* <<administration-security-reads, Read privileges>>
-** <<access-control-privileges-reads-traverse, The `TRAVERSE` privilege>>
-** <<access-control-privileges-reads-read, The `READ` privilege>>
-** <<access-control-privileges-reads-match, The `MATCH` privilege>>
-* <<administration-security-writes, Write privileges>>
-** <<access-control-privileges-writes-create, The `CREATE` privilege>>
-** <<access-control-privileges-writes-delete, The `DELETE` privilege>>
-** <<access-control-privileges-writes-set-label, The `SET LABEL` privilege>>
-** <<access-control-privileges-writes-remove-label, The `REMOVE LABEL` privilege>>
-** <<access-control-privileges-writes-set-property, The `SET PROPERTY` privilege>>
-** <<access-control-privileges-writes-merge, The `MERGE` privilege>>
-** <<access-control-privileges-writes-write, The `WRITE` privilege>>
-** <<access-control-privileges-writes-all, The `ALL GRAPH PRIVILEGES` privilege>>
-* <<administration-security-administration, Security of administration>>
-** <<administration-security-administration-introduction, The `admin` role>>
-** <<access-control-database-administration, Database administration>>
-*** <<access-control-database-administration-access, The database `ACCESS` privilege>>
-*** <<access-control-database-administration-startstop, The database `START`/`STOP` privileges>>
-*** <<access-control-database-administration-index, The `INDEX MANAGEMENT` privileges>>
-*** <<access-control-database-administration-constraints, The `CONSTRAINT MANAGEMENT` privileges>>
-*** <<access-control-database-administration-tokens, The `NAME MANAGEMENT` privileges>>
-*** <<access-control-database-administration-all, Granting `ALL DATABASE PRIVILEGES`>>
-*** <<access-control-database-administration-transaction, Granting `TRANSACTION MANAGEMENT` privileges>>
-** <<access-control-dbms-administration, DBMS administration>>
-*** <<administration-security-administration-dbms-custom, Creating custom roles with DBMS privileges>>
-*** <<access-control-dbms-administration-role-management, The dbms `ROLE MANAGEMENT` privileges>>
-*** <<access-control-dbms-administration-user-management, The dbms `USER MANAGEMENT` privileges>>
-*** <<access-control-dbms-administration-database-management, The dbms `DATABASE MANAGEMENT` privileges>>
-*** <<access-control-dbms-administration-privilege-management, The dbms `PRIVILEGE MANAGEMENT` privileges>>
-*** <<access-control-dbms-administration-execute, The dbms `EXECUTE` privileges>>
-*** <<access-control-dbms-administration-all, Granting `ALL DBMS PRIVILEGES`>>
-* <<administration-built-in-roles, Built-in roles>>
-* <<administration-security-limitations, Known limitations of security>>
-** <<administration-security-limitations-indexes, Security and indexes>>
-** <<administration-security-limitations-labels, Security and labels>>
-** <<administration-security-limitations-db-operations, Security and count store operations>>
-
-
-[[administration-security-introduction]]
-== Introduction
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/index.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/